warehouse/api/grpc.go

```diff
@@ -973,6 +976,24 @@ func (g *GRPC) RetryFailedBatches(
 	return resp, nil
 }
 
+func (g *GRPC) SyncWHSchema(ctx context.Context, req *proto.SyncWHSchemaRequest) (*proto.SyncWHSchemaResponse, error) {
+	log := g.logger.With(
+		lf.DestinationID, req.GetDestinationId(),
+	)
+	log.Infow("Syncing warehouse schema")
+	if req.DestinationId == "" {
+		return &proto.SyncWHSchemaResponse{},
+			status.Errorf(codes.Code(code.Code_INVALID_ARGUMENT), "destinationId cannot be empty")
+	}
+	_, err := g.schemaRepo.SetExpiryForDestination(ctx, req.DestinationId, timeutil.Now())
+	if err != nil {
+		log.Errorw("unable to set expiry for destination", obskit.Error(err))
```

Comment:
Does it need to be a Warning?

Reply:
The API not being able to successfully set expiry should be an error , rt ? Why should it be a warning ?